package.json

```diff
@@ -20,6 +20,7 @@
     "@actions/core": "^1.11.1",
     "@actions/github": "^6.0.0",
     "@ai-sdk/anthropic": "^0.0.56",
+    "@ai-sdk/google": "^0.0.55",
```

Comment:
Ensure that the version of the Google AI SDK (`@ai-sdk/google`) is compatible with the other SDKs in your project. Consider checking the documentation for any breaking changes or required configurations.